Frontend/ui-customer/src/Flow.purs

```diff
@@ -416,6 +433,8 @@ enterMobileNumberScreenFlow = do
                         customerId = ((response.person)^. _id)
                     _ <- lift $ lift $ setLogField "customer_id" $ encode (customerId)
                     setValueToLocalStore CUSTOMER_ID customerId
+                    let _ = setCleverTapUserData "Identity" (show $ getValueToLocalStore CUSTOMER_ID)
```

Comment:
@ashkriti getValueToLocalStore already returns String, so show is not required for this.